docs/resources/service_fortinet.md

```diff
@@ -47,54 +47,61 @@ resource "alkira_fortinet" "test1" {
 
 ### Required
 
-- `credential_id` (String) ID of Fortinet Firewall credential managed by credential resource.
```

Comment:
Could you update your tfplugindoc to latest? There are some newly introduced changes.